presto-docs/src/main/sphinx/language.rst

```diff
@@ -1,9 +1,27 @@
+.. _sql-language:
+
 ************
 SQL Language
 ************
 
+Presto is an ANSI SQL compliant query engine. This standard compliance allows
+Presto user to integrate their favorite data tools, including BI and ETL tools
```

Comment:
"allows the Presto user"
or
"allows Presto users"

Reply:
fixed